packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Is this legit config, or is it just for testing purposes?

Reply:
only for testing purposes, will be deleted before merging